### PR TITLE
added restart flag to service update

### PR DESCRIPTION
**Pull request by argvader**
checking for restart flag on update to enable or disable application restart

### CODE REVIEW

**Comment by bdehamer** [comment]
:+1:

File: app/controllers/services_controller.rb [hunk 1]
```diff
@@ -17,8 +17,9 @@ def create
   end
 
   def update
+    restart = params[:restart].presence =~ /^false$/i ? false : true
```
Comment (bdehamer) [inline app/controllers/services_controller.rb]: Can we just do

    restart = params[:restart] != 'false'

Not sure we really need to worry about case insensitivity

Comment (argvader) [inline app/controllers/services_controller.rb]: sure that will work just fine.. ill update

**Comment by bdehamer** [comment]
:+1:

**Pull request merged**